tests-docker-action/entrypoint.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 git submodule update --init --recursive
 source /opt/ros/foxy/setup.bash
+
+apt-get update
+apt-get install -y gpsd gpsd-clients libgps-dev
```

Comment:
If `gpsd`, `gpsd-clients`, and `libgps-dev` are dependencies now, I think we should probably put them in the README, and also include them in the Docker image rather than installing them at build time on the CI.

Reply:
Personally, I would rather do it this way. Updating the Docker image is kind of difficult, so I think it makes sense for the normal workflow to just edit this CI script. Installing stuff doesn't take much time (since we're not building from source). What do you think?

I'll add this line to the README though, that's a good idea.

Reply:
Yeah, that sounds good. Maybe if we end up having a lot of extra dependencies installed there we can move them to the Docker image all at once? We can cross that bridge when we come to it though.